core/index.ts

```diff
@@ -1,11 +1,12 @@
+// @ts-nocheck
```

Comment:
why do we need this?

Reply:
Good catch – it was for testing, removed!